persistence-cassandra/core/src/test/scala/com/lightbend/lagom/internal/persistence/cassandra/ServiceLocatorHolderSpec.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.persistence.cassandra
+
+import akka.actor.ActorSystem
+import org.scalatest.{ MustMatchers, WordSpec }
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class ServiceLocatorHolderSpec extends WordSpec with MustMatchers {
+  val system = ActorSystem("test")
+
+  "ServiceLocatorHolder" should {
+    "timeout when no service locator is found" in {
+      val eventually = ServiceLocatorHolder(system).serviceLocatorEventually
+      assertThrows[NoServiceLocatorException](
+        Await.result(eventually, 15.seconds)
```

Comment:
This test will always take 10+ seconds to run... if the timeout is made configurable, you could set it lower for the test.